markdown/events/2021/hackathon-march-2021.md

```diff
@@ -83,4 +83,134 @@ More details to come, but so far our idea is to work in the following areas:
+                <!-- <td></td>
+                <td></td>
+                <td></td> -->
+            </tr>
+            <tr>
+                <td data-timestamp="1616410800" data-timeformat="HH:mm z">12:00</td>
+                <!-- <td></td>
+                <td></td>
+                <td></td> -->
+            </tr>
+            <tr>
+                <td data-timestamp="1616412600" data-timeformat="HH:mm z">12:30</td>
+                <!-- <td></td>
+                <td></td>
+                <td></td> -->
+            </tr>
+            <tr>
+                <td data-timestamp="1616414400" data-timeformat="HH:mm z">13:00</td>
+                <td>Group catch-up <br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-dsl2-modules"><i class="fas fa-video mr-1"></i>DSL2-modules</a><br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-pipelines"><i class="fas fa-video mr-1"></i>Pipelines</a><br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-framework"><i class="fas fa-video mr-1"></i>Framework</a><br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-documentation"><i class="fas fa-video mr-1"></i>Documentation</a>
+                </td>
+                <td>Group catch-up <br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-dsl2-modules"><i class="fas fa-video mr-1"></i>DSL2-modules</a><br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-pipelines"><i class="fas fa-video mr-1"></i>Pipelines</a><br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-framework"><i class="fas fa-video mr-1"></i>Framework</a><br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-documentation"><i class="fas fa-video mr-1"></i>Documentation</a>
+                </td>
+                <td>Group catch-up <br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-dsl2-modules"><i class="fas fa-video mr-1"></i>DSL2-modules</a><br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-pipelines"><i class="fas fa-video mr-1"></i>Pipelines</a><br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-framework"><i class="fas fa-video mr-1"></i>Framework</a><br>
+                  <a href="https://meet.jit.si/nf-core-hackathon-march-2021-documentation"><i class="fas fa-video mr-1"></i>Documentation</a>
+                </td>
+            </tr>
+            <tr>
+                <td data-timestamp="1616416200" data-timeformat="HH:mm z">13:30</td>
+                <td>Hack!</td>
+                <td>Hack!</td>
+                <td>Hack!</td>
+            </tr>
+            <tr>
+                <td data-timestamp="1616418000" data-timeformat="HH:mm z">14:00</td>
+                <!-- <td></td>
+                <td></td>
+                <td></td> -->
```

Comment:
What's with the commented out table cells?

Reply:
I wasn't too sure, if I want them in or not. Decided now to not have them in 🙂 .